packages/dialect-react-sdk/hooks/useThreadMessages.ts

```diff
@@ -91,7 +90,7 @@ const useThreadMessages = ({
         (rm) => rm.deduplicationId === lm.deduplicationId
       );
 
-      if (remoteMessage) {
+      if (remoteMessage && lm.deduplicationId) {
```

Comment:
Aren't we finding the remote message by the `deduplicationId`, why do we need to check it here as well? Locally it will always exist

Reply:
agreed, I fixed types of `LocalThreadMessage`